oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreBuilder.java

```diff
@@ -171,6 +171,7 @@ public class DocumentNodeStoreBuilder<T extends DocumentNodeStoreBuilder<T>> {
     private boolean throttlingEnabled;
     private boolean fullGCEnabled;
     private boolean embeddedVerificationEnabled = DocumentNodeStoreService.DEFAULT_EMBEDDED_VERIFICATION_ENABLED;
+    private int fullGCMode;
```

Comment:
please initialize with the default

Reply:
(I see that we're not consistent with this .. eg as with the enabled booleans above, which default to false and that happens to be what is also the default ... perhaps we can ignore that fact plus given this is an `int`, it might be good to set it explicitly)

Reply:
I set it to the default value, thank you for the suggestion.